### PR TITLE
chore: fix button warnings

### PR DESCRIPTION
**Pull request by keita-determined**
## Description

There are some warnings on browser consoles.
`ghost` and `type` cannot be used at the same time.

image

## Test Plan
- Check if all buttons look good and works as expected
- Check if there's no warnings in any pages

## Checklist

- [x] Changes have been manually QA'd
- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [x] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: webui/react/package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "analyze": "source-map-explorer 'build/static/js/*.js'"
   },
   "dependencies": {
+    "@ant-design/icons": "^5.0.1",
```
Comment (keita-determined) [inline webui/react/package.json:34]: we've been using `@ant-design/icons` without explicitly installing it. now its added

File: webui/react/src/pages/WorkspaceDetails/WorkspaceMembers.tsx [hunk 2]
```diff
@@ -222,10 +221,8 @@ const WorkspaceMembers: React.FC<Props> = ({
         title: 'Role',
       },
       {
-        align: 'right',
```
Comment (keita-determined) [inline webui/react/src/pages/WorkspaceDetails/WorkspaceMembers.tsx:225]: Centered action button to follow other tables

File: webui/react/src/components/Metadata/EditableRow.tsx [hunk 3]
```diff
@@ -55,9 +55,11 @@ const EditableRow: React.FC<Props> = ({ name, onDelete, field }: Props) => {
             getPopupContainer={(triggerNode) => triggerNode}
             menu={menu}
             trigger={['click']}>
-            <Button aria-label="action" ghost type="text">
-              <Icon name="overflow-vertical" size="tiny" />
-            </Button>
+            <Button
+              aria-label="action"
+              ghost
+              icon={<MoreOutlined style={{ fontSize: '20px' }} />}
```
Comment (julian-determined-ai) [inline webui/react/src/components/Metadata/EditableRow.tsx:61]: @keita-determined Why the change to `MoreOutlined` with styling instead of:


```
<Button
 icon={<Icon name="overflow-vertical" size="tiny" />}
 </Button>
```

?

Comment (keita-determined) [reply webui/react/src/components/Metadata/EditableRow.tsx:61]: thats why PR needs to be reviewed. you are right. i was thinking about something else.
i'll replace them all with `<Icon />`

Comment (keita-determined) [reply webui/react/src/components/Metadata/EditableRow.tsx]: fixed

File: webui/react/package.json (hunk 1 above)
Comment (julian-determined-ai) [reply webui/react/package.json:34]: @keita-determined It looks like we were using `4.7.0` and we are now defining version `5.0.1` any explicit reason for the version change?

Comment (keita-determined) [reply webui/react/package.json]: i dont think version really matters because npm nicely adjust version, but i changed it with v4.7.0

**Review by julian-determined-ai — CHANGES_REQUESTED** [review]
I am hesitant to add another dependency to our `package.json` unless explicitly needed. 

```
    "node_modules/antd": {
      ....
      "dependencies": {
        "@ant-design/icons": "^4.7.0",
```

the icons are added as a dependency via `antd`, which can be seen in our `package-lock.json`. In theory this change would be needed if we were not using `antd` and still wanted the icon set. But I don't see that being the case in the near future. 

If there is some reason you think we need this change please let me know.

**Comment by keita-determined** [comment]
@julian-determined-ai ok i removed packges.json changes. only button warning fixes.

**Review by julian-determined-ai — APPROVED** [review]
LGTM!
